frontend/src/components/pages/CampOverview/EditCamperModal/EditCamperFormResponseSection.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+
+import {
+  VStack,
+  RadioGroup,
+  Stack,
+  Radio,
+  Textarea,
+  CheckboxGroup,
+  Checkbox,
+  FormLabel,
+  FormControl,
+} from "@chakra-ui/react";
+import { FormQuestion } from "../../../../types/CampsTypes";
+
+const EditCamperFormResponseSection = ({
+  formQuestions,
+  camperResponses,
+  setFormResponses,
+}: {
+  formQuestions: FormQuestion[];
+  camperResponses: Map<string, string>;
+  setFormResponses: (formResponses: Map<string, string> | undefined) => void;
+}): JSX.Element => {
+  const updateFormResponse = (question: string, option: string) => {
+    const newCamperResponse = new Map(Object.entries(camperResponses));
+    newCamperResponse.set(question, option);
+    setFormResponses(newCamperResponse);
+  };
+
+  const renderMCQ = (
+    question: string,
+    answer: string | undefined,
+    options: string[],
+    questionId: string,
+    isRequired: boolean,
+  ) => {
+    return (
+      <FormControl key={questionId} isRequired={isRequired}>
+        <FormLabel>
+          <b>{question}</b>
+        </FormLabel>
+        <RadioGroup defaultValue={answer}>
+          <Stack direction="column">
+            {options.map((option) => (
+              <Radio
+                key={`${questionId}_${option}`}
+                value={option}
+                onChange={() => {
+                  updateFormResponse(question, option);
+                }}
+              >
+                {option}
+              </Radio>
+            ))}
+          </Stack>
+        </RadioGroup>
+        <br />
```

Comment:
Instead of `<br>` add lower padding to the element above (in this case `<RadioGroup>`, same comment whereever else you use `<br>`s to avoid pure HTML elements.

Reply:
done